### PR TITLE
[MIG][15.0] product_fao_fishing: Migration to v15.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT36496

ping @carlosdauden @CarlosRoca13

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: product_fao_fishing/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+#~ msgid "Draft nets"
+#~ msgstr "Borrador"
+
+#, fuzzy
+#~ msgid "Left Parent"
+#~ msgstr "Padre izquierdo"
+
+#, fuzzy
+#~ msgid "Right Parent"
+#~ msgstr "Padre derecho"
```
Comment (CarlosRoca13) [inline product_fao_fishing/i18n/es.po]:
```suggestion

```

**Review by CarlosRoca13 — COMMENTED** [review]
Should we make a migration script to convert the data set on `product.fao.fishing.technique` to set it on `product.attribute.values`?

**Comment by sergio-teruel** [comment]
@CarlosRoca13 changes done

**Review by rafaelbn — APPROVED** [review]
Functional review 👍 

imagen

**Comment by pedrobaeza** [comment]
> Should we make a migration script to convert the data set on `product.fao.fishing.technique` to set it on `product.attribute.values`?

@sergio-teruel what about this?

**Comment by sergio-teruel** [comment]
@pedrobaeza @CarlosRoca13 Clean translations and migration scripts done.. 
Maybe the pre-migration file could be removed

**Comment by sergio-teruel** [comment]
ping  @carlosdauden

**Review by pedrobaeza — APPROVED** [review]
Code review

**Pull request merged**